tests/properties/worthless_test.php

```diff
@@ -69,7 +70,7 @@ public function tearDown() {
      * @access public
      */
     public function test_propertiesCount() {
-        $this->assertCount(28, $this->properties);
+        $this->assertCount(29, $this->properties);
```

Comment:
This count should be solved over an property. Also the property array only in my view.

Reply:
I totally agree, the count is 100 % worthless - as the file name suggests 😊 I'll rename the test to PublicInterfaceTest.